pkg/devicetwin/dtcontroller.go

```diff
@@ -77,8 +77,9 @@ func (dtc *DTController) Start() error {
 			if msg, ok := dtc.DTContexts.ModulesContext.Receive("twin"); ok == nil {
 				log.LOGGER.Info("DeviceTwin receive msg")
 				err := dtc.distributeMsg(msg)
-				// todo  deal error
 				if err != nil {
+					log.LOGGER.Infof("distributeMsg failed: %v", err)
+					return
```

Comment:
We cannot have a return here.... this is an infinite loop here to receive messages for twin module...
If we return, device twin will stop receiving messages. My bad i missed it earlier

Reply:
Updated